.vscode/settings.json

```diff
@@ -30,5 +30,11 @@
     "**/CVS": true,
     "**/.DS_Store": true,
     "**/*.pyc": true
-  }
+  },
+  "workbench.colorTheme": "Monokai",
+  "python.testing.promptToConfigure": false,
+  "python.testing.unittestEnabled": false,
+  "python.testing.nosetestsEnabled": false,
+  "python.pythonPath": "C:\\Users\\t-susoti\\AppData\\Local\\Programs\\Python\\Python37\\python.exe",
```

Comment:
Do not commit changes in this file, especially not a path of your machine :)

Reply:
done